articles/forest-offsets-explainer/faq.md

```diff
@@ -111,27 +109,23 @@ One of the important features of the FIA database is that it changes over time.
 
 For our new study, we used a contemporary version of the FIA database. This does not mean that we used “newer” data reporting forest carbon levels during years that weren’t part of CARB’s calculations. Instead, we relied only on older FIA ‘inventories’ in the larger FIA database — i.e. the same sets of plot measurements. As described in our extended methods:
 
-<Blockquote source={'Extended Methods / page 6'}>
```

Comment:
Ah! It was, because in standardizing on a `Blockquote` component I didn't want to try to incorporate a source into the component itself, as that's actually quite hard to do via generic markdown formatting (where do you put the argument), and we can always specify this kind of thing in the adjacent text. But, I forgot to do that! So I just made sure the page numbers are referenced. I honestly prefer it this way too, as I never quite liked the special treatment for source anyway.